src/components/course/CourseSidebarPrice.jsx

```diff
@@ -48,13 +48,23 @@ const CourseSidebarPrice = () => {
     );
   }
 
-  // Case 2: No subsidies found
   const hasDiscountedPrice = coursePrice.discounted < coursePrice.list;
+  // Case 3: No subsidies found but Browse and Request Enabled
+  if (!hasDiscountedPrice && features.FEATURE_BROWSE_AND_REQUEST) {
```

Comment:
Per the AC this is behind a feature flag. I do think this technically should be behind an enterprise configuration instead, but that doesn't exist yet.

Reply:
I agree, this should be a configuration but we are not there yet. I've filed a ticket here to cover this change: https://openedx.atlassian.net/browse/ENT-5380